source/includes/acts/get_api.md

```diff
@@ -156,7 +239,7 @@ Attribute | Type | Description
 --------- | ---- | -----------
 PageNumber | int | Page number of the results to fetch.
 PageSize | int | The number of results per page
-emissionCategoryID | int | Emission Category ID is one of the unique identifier for this emission category record and is one of the filter option provided for Emission Category
+emissionCategoryIDs | int | Emission Category ID is one of the unique identifier for this emission category record and is one of the filter option provided for Emission Category
```

Comment:
Emission Category ID is the unique identifier for an emission category record and is one of the filter options provided for Emission Category